### ISSUE TITLE
recreate pretrained model

### ISSUE THREAD

**Issue by yondu22** [NVIDIA/tacotron2#267]
Do you have the exact steps to recreate the published model that achieved lowest validation loss? the hyperparams, the step to anneal learning rate, any other tricks

I can't repeat the process to create the pretrained model by just using train.py

**Comment by rafaelvalle** [comment]
Closing due to inactivity.

**Comment by thangdc94** [comment]
I also want to create a pretrained model to boost up my training process. Can you give some advices?
